edm4eic.yaml

```diff
@@ -275,8 +282,23 @@ datatypes:
       - int32_t           sector            // The sector this hit occured in
       - edm4hep::Vector3f local             // The local position of the hit in detector coordinates (relative to the sector) [mm]
 
+  edm4eic::CherenkovParticleID:
+    Description: "Cherenkov detector PID"
+    Author: "A. Kiselev, C. Chatterjee, C. Dilks"
+    Members:
+      - int32_t           radiator          // Radiator number
+      - float             theta             // Estimated average Cherenkov cone angle
```

Comment:
This is observed, not estimated, right? If estimated, doesn't it depend on the particle type hypothesis?

Isn't this just avg of the vector members and therefore redundant?

Reply:
> This is observed, not estimated, right? If estimated, doesn't it depend on the particle type hypothesis?

Photon theta is estimated from the ray-tracing, independent of hypothesis. 

> Isn't this just avg of the vector members and therefore redundant?

Correct, it is redundant and we can remove this member.